test/integration/api/integrations_controller_test.rb

```diff
@@ -173,12 +172,13 @@ def test_update_proxy_rule_position
     put admin_service_integration_path(service_id: service.id), proxy: {api_backend: '1'}
   end
 
-  def test_edit
+  # Regression test for pre-APIAP
+  def test_edit_not_found
     get edit_admin_service_integration_path(service_id: 'no-such-service')
     assert_response :not_found
 
     get edit_admin_service_integration_path(service_id: service.id)
-    assert_response :success
+    assert_response :not_found
   end
```

Comment:
This test is completely removed in https://github.com/3scale/porta/pull/2288, plz do not change it here as well 😄

Reply:
It will break the build otherwise 🤷‍♂️